### PR TITLE
Misc hooks fixes

### PR DESCRIPTION
**Pull request by mbouron-gpsw**

### CODE REVIEW

File: pynodegl-utils/pynodegl_utils/hooks.py [hunk 1]
```diff
@@ -52,14 +52,17 @@ def get_sessions(self):
...
-            ))
+            try:
+                # TODO: asynchronous calls? (individual session queries could take some time)
+                session_info = self._module.get_session_info(session_id)
+                sessions.append((
+                    session_id,
+                    session_desc,
+                    session_info.get('backend', ''),
+                    session_info.get('system', ''),
+                ))
```
Comment (ubitux) [inline pynodegl-utils/pynodegl_utils/hooks.py:63]: Can you keep this part out of the try catch? Same in the other commit.

LGTM otherwise, thanks

Comment (mbouron-gpsw) [reply pynodegl-utils/pynodegl_utils/hooks.py]: Done. I used a continue in the `except` block. Not sure if that's the preferred form.

Comment (cboesch-gpsw) [reply pynodegl-utils/pynodegl_utils/hooks.py]: Thanks; I would have picked that form as well to reduce the indent level for the valid codepath

**Review by cboesch-gpsw — APPROVED** [review]

File: pynodegl-utils/pynodegl_utils/hooks.py (hunk 1 above)
Comment (mbouron-gpsw) [reply pynodegl-utils/pynodegl_utils/hooks.py]: Thanks !

**Pull request merged**